conf.yaml

```diff
@@ -81,3 +82,67 @@ repository:
                     comment: |
                         {author}'s PR has reached the required number of votes
                         for merging.
+        -
+            name: Check merged PRs for review tag.
+            conditions:
+                - state: 'merged'
+                - created_at__ge: 2016
+            actions:
+                -
+                    action: assert_has_tag
+                    action_value: 'status/review'
+                    comment: |
+                        This PR was merged without a 'status/review' tag, please correct.
+        -
+            name: Check merged PRs for a kind tag.
+            conditions:
+                - state: 'merged'
+                - created_at__ge: 2016
+            actions:
+                -
+                    action: assert_has_tag__startswith
+                    action_value: 'kind/'
+                    comment: |
+                        This PR was merged without a 'kind/' tag, please correct.
+        -
+            name: Add status/review to new PRs.
+            conditions:
+                - state: 'merged'
+                - created_at__ge: '2016-01-01'
```

Comment:
This ``created_at__ge`` is just so only new PRs get tagged. All new PRs should gt tagged with status/review at first, not just ones in this particular release. This condition will never have to be updated - I just don't want to assign status/review to PR #1...750 or whatever.